src/quo2/components/dividers/strength_divider/style.cljs

```diff
@@ -0,0 +1,13 @@
+(ns quo2.components.dividers.strength-divider.style)
+
+(def container
+  {:height             40
+   :flex-direction     :row
+   :padding-horizontal 20
+   :align-items        :center})
+
+(defn text
+  [color]
+  {:color       color
+   :margin-left 5})
```

Comment:
I think the space between the components is 4 instead.
![Screenshot 2023-02-24 at 15-50-36 Figma](https://user-images.githubusercontent.com/90291778/221299996-a5b81ce1-2c88-4c9f-b33d-38439c39edd1.png)